src/lib.rs

```diff
@@ -127,7 +127,7 @@ impl KISSContext {
         CONTEXT.with(|context|
             context.borrowed_strs.borrow_mut()
                 .entry(widget).or_insert_with(HashMap::new)
-                .entry(str_).or_insert_with(|| Rc::new(Cell::new(0)))
+                .entry(str_).or_insert_with(|| Rc::new(Cell::new(1)))
```

Comment:
This needs to be initialized to zero, because it also needs to be incremented unconditionally after being returned from this expression.